lib/compiler/swc/swc-compiler.ts

```diff
@@ -170,6 +171,21 @@ export class SwcCompiler extends BaseCompiler {
       swcOptions.jsc.baseUrl = path.join(rootDir, swcOptions.jsc.baseUrl);
     }
 
+    if (swcOptions?.jsc?.paths) {
```

Comment:
This condition will always be true for someone who has paths defined in the tsconfig.json file (which is required for DX)

Reply:
Yes, `swcRcFile?.jsc?.paths` will throw error when paths didn't defined in `.swcrc`. This is the `@swc/cli` BUG in Windows, if pass an object to `swcCli.default` function.